spec/spec_helper_acceptance.rb

```diff
@@ -4,7 +4,19 @@
 unless ENV['BEAKER_provision'] == 'no'
   hosts.each do |host|
     # Install Puppet
-    if host.is_pe?
+    if ENV['SHA']
+      install_puppetlabs_release_repo(host)
+      install_puppetlabs_dev_repo(host, 'puppet', ENV['SHA'])
+
+      #install_packages_from_local_dev_repo(host, 'puppet')
+      if host['platform'] =~ /debian|ubuntu|cumulus/
+        on host, 'apt-get install -y puppet'
+      elsif host['platform'] =~ /fedora|el|centos/
+        on host, 'yum install -y puppet'
```

Comment:
At some point do we want to install/configure MCO so that we can verify migration is correct there?